pulp_container/app/checks.py

```diff
@@ -0,0 +1,43 @@
+from django.conf import settings
+from django.core.checks import Error as CheckError, register
+
+
+@register(deploy=True)
+def container_settings_check(app_configs, **kwargs):
+    errors = []
+
+    # Other checks only apply if token auth is enabled
+    if str(getattr(settings, "TOKEN_AUTH_DISABLED", False)).lower() == "true":
+        return errors
+
+    if getattr(settings, "TOKEN_SERVER", None) is None:
+        errors.append(
+            CheckError(
+                "TOKEN_SERVER variable has to be set when token authentification is enabled",
```

Comment:
```suggestion
                "TOKEN_SERVER is a required setting that has to be configured when token authentification is enabled.",
```

Same for the others. Do you like it more?